Documentation/gettingstarted/istio.rst

```diff
@@ -93,15 +99,30 @@ Deploy Istio on Kubernetes:
 
     $ kubectl create -f ${ISTIO_HOME}/install/kubernetes/istio.yaml
 
-Check the progress of the deployment (every services should have an
+.. TODO:
+   Rewrite istio.yaml to replace the proxy and proxy_init images with
+   Cilium's images.
+
+Configure Istio's sidecar injection to use Cilium's Docker images for the
```

Comment:
This needs an explanation that we a replacing the images in order to allow for Istio and Cilium to share the Envoy proxy.

Reply:
That’s not yet true. I was thinking of writing this up when we’re really running the L7 filter in the sidecar.

Reply:
OK, makes sense